recipes/libpng/all/conanfile.py

```diff
@@ -1,7 +1,10 @@
-from conans import ConanFile, CMake, tools
 import os
+from conan import ConanFile
+from conans import tools, CMake
+from conan.tools.files import apply_conandata_patches
```

Comment:
```suggestion
from conan.tools.files import apply_conandata_patches, get, rmdir
```
The linter wants new tools. 

Need to update tools.get( to `get(self, ...`

Same for rmdir